packages/grub2/config/grub.cfg

```diff
@@ -1,12 +1,26 @@
 set timeout=10
 
+# Load custom env file
 set env_file="/grubenv"
 search --file --set=env_blk "${env_file}"
 
+# Load custom env file
+set oem_env_file="/grub_oem_env"
```

Comment:
The rebranding logic writes in this file instead of `/grubenv` or `/grubmenu`. Those are left for users to specify, while during install we instrument only `/grub_oem_env`